src/ochubryk/Hometask2/fibonacci.py

```diff
@@ -0,0 +1,17 @@
+n = int(input('Введите порядковый номер числа Фибоначчи: '))
+f1 = f2 = f3 = 1
+
+if n < 2:
+    print('Первый элемент ряда Фибоначчи: 1')
+else:
+    print('Числа ряда Фибоначчи до искомого элемента(включительно): ')
+    print(f1, end=' ')
+    print(f2, end=' ')
+
+    for i in range(2, n):
+        f3 = f1 + f2
```

Comment:
f1, f2 = f1+f2, f1 
in this way we can use only 2 temporary variables

Reply:
Fixed